docs/modules/ROOT/pages/ios_mdm.adoc

```diff
@@ -9,15 +9,610 @@
 
 Starting with iOS 7, Apple added support for {mdm-protocol-ref-url}[managed application configuration]. 
 An MDM server can push a configuration to the iOS App. 
-The app can access the configuration using the `NSUserDefaults` class. 
+The app can access this configuration (read-only) using the `NSUserDefaults` class by reading a configuration dictionary under the key _com.apple.configuration.managed_. An app can also observe a system notification (_NSUserDefaultsDidChangeNotification_) to get notified about configuration changes. In addition feedback can be queried back by MDM server. To enable that, app has to write a dictionary with feedback information into user defaults under _com.apple.feedback.managed_ key.
 The configuration is basically a key-value dictionary provided as a `.plist` file.
 
+=== Configurable Settings
+
+ownCloud App implements a mechanism internally called Class Settings which can be derived from different sources:
+
+- Environment variables which e.g. can be set in Xcode for testing. In this case setting keys have to be prepended with _oc:_ prefix.
+- User preferences accessed by the very same API but stored under _org.owncloud.user-settings_ key.
+- Settings dictionary pushed by an MDM Server and accessible using `NSUserDefaults` API under the key _com.apple.configuration.managed_.
+- Default settings defined directly in the app sourcecode.
```

Comment:
Also mention, that `Branding.plist` now also can provide values to ClassSettings from a `Configuration` section?

(from https://github.com/owncloud/ios-app/pull/797)

Reply:
@michaelstingl just added this